__tests__/app.test.js

```diff
@@ -379,6 +380,7 @@ describe("PATCH /api/articles/:article_id", () => {
         const {
           body: { article },
         } = await request(app).get(`/api/articles/${testArticleId}`);
+        delete article.comment_count;
```

Comment:
what's this line doing?

Reply:
It's due to the way I was testing the accuracy of the return value in the patch endpoint: by comparing the updated value returned in the response, to one subsequently called directly from the db. The reason I then deleted this property from the object prior to comparing it, is that the feature (in this pr) changed the endpoint get/api/articles to add a comment_count to the response when called, but this is not present in the article table itself, so would result in a false positive in the prior comparison logic, without the amendment made on line 383.